app/routes/_layout+/calendar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import FullCalendar from "@fullcalendar/react";
+import { json, redirect } from "@remix-run/node";
+import type { LoaderFunctionArgs } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { Breadcrumbs } from "~/components/layout/breadcrumbs";
+import Header from "~/components/layout/header";
+import type { HeaderData } from "~/components/layout/header/types";
+import Heading from "~/components/shared/heading";
+import { tw } from "~/utils/tw";
+export const loader = () => {
+  const header: HeaderData = {
+    title: "Calendar", // Change to an appropriate title
+  };
+
+  return json({ header });
+};
+const Calendar = () => {
+  const { header } = useLoaderData<typeof loader>();
+  return (
+    <>
+      <header className={tw("-mx-4 bg-white")}>
```

Comment:
Just use <Header/> component and pass it title and subtitle. That will render it in our normal styles.

Reply:
Made the changes, used the already existing Header component.